samples/peripheral/radio_test/README.rst

```diff
@@ -0,0 +1,189 @@
+.. _radio_test:
+
+Radio Test
+##########
+
+The Radio Test sample demonstrates how to configure the radio in a specific mode and then test its performance.
+The sample provides a set of predefined commands that allow you to configure the radio in three modes:
+
+* Constant RX or TX carrier
+* Modulated TX carrier
+* RX or TX sweep
+
+Overview
+********
+
+The tests are controlled with Command Line Interface (CLI) through the serial port.
+Zephyr's :ref:`zephyr:shell_label` module is used to handle the commands.
+At any time during the tests, you can dynamically set the radio parameters, such as output power, bit rate, and channel.
+In sweep mode, you can set the time for which the radio scans each channel from 1 millisecond to 99 milliseconds, in steps of 1 millisecond.
+The sample also allows you to send a data pattern to another board.
+
+The sample starts with enabling the high frequency crystal oscillator and configuring CLI.
+You can then start running commands to set up and control the radio.
+See :ref:`radio_test_ui` for a list of available commands.
+
+.. note::
+   For the IEEE 802.15.4 mode, the start channel and the end channel must be within the channel range of 11 to 26.
+   Use the ``start_channel`` and ``end_channel`` commands to control this setting.
+
+Requirements
+************
+
+* One of the following development boards:
+
+  * |nRF5340DK|
+
+    * Network Core
+
+  * |nRF52840DK|
+  * |nRF52DK|
+
+* One of the following testing devices:
+
+  * Another board with the same sample.
+    See :ref:`radio_test_testing_board`.
+  * Another board connected to a PC with RSSI Viewer application (available in the `nRF Connect for Desktop`_).
+    See :ref:`radio_test_testing_rssi`.
+
+.. note::
+   The radio test can be also performed using a spectrum analyzer.
+   This method of testing is not covered by this documentation.
+
+.. _radio_test_ui:
+
+User interface
+**************
+.. list-table:: Main CLI commands (in alphabetical order)
```

Comment:
Still doubtful about the CLI here ;)

Reply:
Are you suggesting that I should use Shell instead of CLI? (it's Shell subsystem in Zephyr after all)  Or what do you mean?

Reply:
Yes, "Main shell commands ..." works.